### PR TITLE
STONEBLD-27: continue cleaning up the rest related to removed build-pipelines-defaults configmap

### PR DESCRIPTION
**Pull request by tkdchen**

### CODE REVIEW

File: components/build-service/README.md [hunk 1]
```diff
@@ -47,24 +47,10 @@ MY_QUAY_USER=mkovarik PIPELINESASCODE=1 ./hack/build/build-via-appstudio.sh http
...
-```
-oc create configmap build-pipelines-defaults --from-literal default_build_bundle=$BUNDLE
-```
-
-## HACBS enablement
-
-HACBS workflow can be set by creation of configmap `hacbs` in the user namespace.
 
-`oc create configmap hacbs` will:
+By default the bundle is defined in `components/build-service/build-pipeline-selector.yaml` namespace.
```
Comment (mmorhun) [inline components/build-service/README.md:51]: You refer to a `build-pipeline-selector` object here, but the sentence ends with word `namespace`.

File: components/build-service/README.md [hunk 2]
```diff
@@ -47,24 +47,10 @@ MY_QUAY_USER=mkovarik PIPELINESASCODE=1 ./hack/build/build-via-appstudio.sh http
...
 
-1. Always use Pipelines-as-Code
-2. Selects bundle from `hacbs_build_bundle` key in configmap `build-pipelines-defaults`
+To change the default pipeline bundles, either update the default yaml file and provision cluster again, or create a `BuildPipelineSelector` CR in your own namespace. Refer to the `build-pipeline-selector.yaml` as an example.
```
Comment (mmorhun) [inline components/build-service/README.md:53]: > update the default yaml

It's not clear where is that default yaml file and what to change in it.

**Comment by Michkov** [comment]
Timeouts increased in e2e-tests

/test appstudio-e2e-tests

**Pull request merged**